infoblox_client/connector.py

```diff
@@ -25,8 +25,15 @@
 except ImportError:
     import urllib.parse as urlparse
 
-from oslo_log import log as logging
-from oslo_serialization import jsonutils
+try:
+    from oslo_log import log as logging
+except ImportError:
```

Comment:
Missing 
```py
except ImportError:  # pragma: no cover
```

Reply:
thank you very much for your patience
commit cd0f4b1f9d018a6a98c103370761c05a4295ad26 (HEAD -> master, origin/master, origin/HEAD)
Author: Markus Mueller <markus.mueller@wirecard.com>
Date:   Wed Feb 27 22:08:50 2019 +0100

     add codecov no-pragma for import excepts - also when needed twice